ee/codegen/src/__test__/nodes/__snapshots__/api-node.test.ts.snap

```diff
@@ -94,6 +94,84 @@ class APINode(BaseAPINode):
 "
 `;
 
+exports[`ApiNode > basic api node > should generate API key header fields when API key is provided 1`] = `
+"from vellum.workflows.constants import APIRequestMethod, AuthorizationType
+from vellum.workflows.nodes.displayable import APINode as BaseAPINode
+
+
+class APINode(BaseAPINode):
+    url = "fasdfadsf"
+    method = APIRequestMethod.GET
+    api_key_header_key = "X-API-KEY"
+    authorization_type = AuthorizationType.API_KEY
+    api_key_header_value = "my-api-key"
+"
+`;
+
+exports[`ApiNode > basic api node > should generate all fields when all conditions are provided 1`] = `
+"from vellum.workflows.constants import APIRequestMethod, AuthorizationType
+from vellum.workflows.nodes.displayable import APINode as BaseAPINode
+
+
+class APINode(BaseAPINode):
+    url = "fasdfadsf"
```

Comment:
[nit] let's use something that looks like a url

oh looks like we copied this from `apiNodeFactory` - let's have both use something url-looking